README.md

```diff
@@ -1,78 +1,36 @@
 # DevTools RFCs
 
-Most changes to our tools, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow.
+Some changes to DevTools need feedback from the team and community to reach a consensus. In such a case you should use this repository to discuss the cahnge proposal proposal.
 
-Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the [DevTools core team][].
+DevTools RFC process aims to be lightweight and focused on conversations. Therefore opening a DevTools RFC is as easy as opening an issue in this repository to discuss the proposal.
 
-The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the framework.
+The [DevTools core team][] will be ultimately responsible for accepting or rejecting a change proposal, but the discussion is open to everybody.
 
-[Active RFC List][]
+The list of RFCs currently under discussion can be found at: [Active RFC List][].
 
-## When you need to follow this process
+## Creating a RFC
 
-You need to follow this process if you intend to make "substantial" changes to DevTools or its documentation. What constitutes a "substantial" change is evolving based on community norms, but may include the following.
+* Create an issue with the "proposal" label
```

Comment:
Do we want to suggest / force some structure to the issue? I like the idea of keeping it lightweight but maybe a small issue template would make things easier to structure a proposal. I'd also be OK to not do this now and adapt the process based on common problems that arise in issues that get filed.

Reply:
That's exactly how I would like to proceed. I do believe that a template/model/set of guidelines for creating the issue would be beneficial. But I think it's better to wait for the few first issues to see what seems to work :)